misc/ballerina-formatter/src/main/java/org/ballerinalang/format/FormatUtil.java

```diff
@@ -403,7 +403,7 @@ private static void generateChangeReport(List<String> formattedFiles, boolean dr
     }
 
     private static String getModuleName(String moduleName) {
-        String[] splitedTokens = moduleName.split(File.separator);
+        String[] splitedTokens = moduleName.split("" + File.separator);
```

Comment:
Is it safe to use `File.separator` here?
What about in windows env?

Reply:
This will work in both Unix and windows environment...